sklego/meta/_decay_utils.py

```diff
@@ -0,0 +1,272 @@
+import numpy as np
```

Comment:
Should we add check for `min_value` to be strictly positive? Estimators will handle that anyway, I would say it is redundant